tool/tsh/tsh.go

```diff
@@ -398,7 +398,7 @@ func onLogin(cf *CLIConf) {
 		// but cluster is specified, treat this as selecting a new cluster
 		// for the same proxy
 		case (cf.Proxy == "" || host(cf.Proxy) == host(profile.ProxyURL.Host)) && cf.SiteName != "":
-			tc.SaveProfile("")
+			tc.SaveProfile("", "")
```

Comment:
`SaveProfile` should take a struct of params, `SaveProfile("", "")` is not clear what's being saved. This would also simply things when no alias is needed (see other comment), because you could do things like:

```go
tc.SaveProfile(&profile{
   path: "/path/to/profile",
})
```